install-prereqs.sh

```diff
@@ -30,6 +31,18 @@ sh get-docker.sh
 groupadd docker || true
 usermod -aG docker $USER || true
 
+# Install kubectl and gettext - needed for k8s-* backends
+cat <<EOF > /etc/yum.repos.d/kubernetes.repo
```

Comment:
you are aware we aren't using this anymore, you, might supply instruction in the README.rst for ubuntu 
if someone would like to debug some thing here localy

Reply:
`README.rst` still has the instruction to run `install-prereqs.sh` for local SCT setup. So, this change is simply out of scope of this PR#.